### PR TITLE
30 create hero section

### PR DESCRIPTION
**Pull request by ahmedirheem**
related #30

### CODE REVIEW

File: client/src/components/hero/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { JoinButton, Wrapper } from './HeroSection.styled';
+import HeroDescription from './HeroDescription';
+
+interface Props {
+  image: string;
+  type: string;
+}
+
+const Hero: FC<Props> = ({ image, type }) => {
```
Comment (alaaahmade) [inline client/src/components/hero/Hero.tsx:10]: Don't forget to add the type of returning function : ```: ReactElement```

File: client/src/components/hero/HeroDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { Description, DescriptionWrap, Title } from './HeroSection.styled';
+
+interface Props {
+  title: string;
+  description: string;
+}
+
+const HeroDescription: FC<Props> = ({ title, description }) => {
```
Comment (alaaahmade) [inline client/src/components/hero/HeroDescription.tsx:9]: Don't forget to add the type of returning function : ```: ReactElement```

File: client/src/components/hero/HeroSection.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  imageUrl: string;
+}
+export const Wrapper = styled('div')<Props>({
+  width: '100%',
+  height: '100%',
+  cursor: 'pointer',
+  border: '2px solid #01031c',
+  flexGrow: '1',
+  position: 'relative',
+  backgroundImage:
```
Comment (Saeed99Madi) [inline client/src/components/hero/HeroSection.styled.tsx:91]: you have installed the images, why using another one !

**Review by Saeed99Madi — CHANGES_REQUESTED** [review]

File: client/src/components/hero/HeroSection.styled.tsx (hunk 3 above)
Comment (Heba-Adwan) [reply client/src/components/hero/HeroSection.styled.tsx:91]: solve conflict

**Pull request closed without merging**